apache2/waf_lock/murmur3.c

```diff
@@ -0,0 +1,319 @@
+/* Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why do we need the murmur hash?

Reply:
it seems to be an overkill, i will just let user input the sem id

Reply:
+1